patching_tool_version.sh

```diff
@@ -19,13 +20,13 @@ function get_full_version {
 		echo "1.0.24"
 	elif [ "${1}" == "2.0" ]
 	then
-		echo "2.0.16"
+		echo $(lc_curl https://releases.liferay.com/tools/patching-tool/LATEST-2.0.txt)
 	elif [ "${1}" == "3.0" ]
 	then
-		echo "3.0.38"
+		echo $(lc_curl https://releases.liferay.com/tools/patching-tool/LATEST-3.0.txt)
 	elif [ "${1}" == "4.0" ]
 	then
-		echo "4.0.3"
+		echo $(lc_curl https://releases.liferay.com/tools/patching-tool/LATEST-4.0.txt)
```

Comment:
Hey @lucasmiranda0, your fix makes sense to me though do we really need to update the Patching Tool version when building the Docker images since the latest version is already being added to the portal bundle?

Please reach me out off-line tomorrow for we discuss about it.

Thanks